contrib/hack/kubectl

```diff
@@ -34,7 +34,7 @@ if [ -n "${PORT}" ]; then
 	SERVER_INFO="--server localhost:${PORT}"
 fi
 
-docker run --rm -ti --net host \
```

Comment:
Not sure this is right. If kubectl ever needs input from the user (e.g. kubectl run --image ubuntu bash) then w/o `-ti` things won't work.